kodilla-patterns/src/main/java/com/kodilla/patterns/factory/tasks/TaskFactory.java

```diff
@@ -0,0 +1,20 @@
+package com.kodilla.patterns.factory.tasks;
+
+public final class TaskFactory {
+    public static final String SHOPPING = "SHOPPING";
+    public static final String PAINTING = "PAINTING";
+    public static final String DRIVING = "DRIVING";
+
+    public final Task createTask(final String taskClass) throws IllegalArgumentException {
+        switch (taskClass) {
```

Comment:
a jesli chcialbym utworzyc zadanie "Paining walls", "white paint", 1) to jak takie zadanie mi utworzy ta fabryka?